daprdocs/content/en/operations/hosting/kubernetes/kubernetes-dapr-shared.md

```diff
@@ -0,0 +1,86 @@
+{{% /alert %}}
+
+
+### `DeamonSet` strategy
+
+With Kubernetes `DaemonSet`, you can define workloads that need to be deployed once per node in the cluster. This enables workloads that are running on the same node to communicate with local Dapr APIs, no matter where the Kubernetes `Scheduler` schedules your workload.
+
+<img src="/images/dapr-shared/daemonset.png" width=800 style="padding-bottom:15px;">
+
+{{% alert title="Note" color="primary" %}}
+Since `DaemonSet` installs one instance per node, it consumes more overall resources in your cluster, compared to `Deployment` for a per cluster deployment.
+{{% /alert %}}
+
+
+### `Deployment` strategy
+
+Kubernetes `Deployments` are installed once per cluster. Based on available resources, the Kubernetes `Scheduler` decides on which node the workload is scheduled. For Dapr Shared, this means that your workload and the Dapr instance might be located on separate nodes, which can introduce considerable network latency with the trade-off of reduce resource usage.
+
+<img src="/images/dapr-shared/deployment.png" width=800 style="padding-bottom:15px;">
+
+## Getting Started with Dapr Shared
+
+{{% alert title="Prerequisites" color="primary" %}}
+Before installing Dapr Shared, make ensure you have [Dapr installed in your cluster]({{< ref "kubernetes-deploy.md" >}}).
+{{% /alert %}}
+
+If you want to get started with Dapr Shared, you can create a new Dapr Shared instance by installing the official Helm Chart:
+
+```
+helm install my-shared-instance oci://registry-1.docker.io/daprio/dapr-shared-chart --set shared.appId=<DAPR_APP_ID> --set shared.remoteURL=<REMOTE_URL> --set shared.remotePort=<REMOTE_PORT> --set shared.strategy=deployment
+```
+
+Your Dapr-enabled applications can now make use of the Dapr Shared instance by pointing the Dapr SDKs to or sending requests to the `my-shared-instance-dapr` Kubernetes service exposed by the Dapr Shared instance. 
+
+> The `my-shared-instance` above is the Helm Chart release name. 
+
+If you are using the Dapr SDKs, you can set the following environment variables for your application to connect to the Dapr Shared instance (in this case, running on the `default` namespace): 
+
+```
+        env:
+        - name: DAPR_HTTP_ENDPOINT
+          value: http://my-shared-instance-dapr.default.svc.cluster.local:3500
+        - name: DAPR_GRPC_ENDPOINT
+          value: http://my-shared-instance-dapr.default.svc.cluster.local:50001 
+```
+
+If you are not using the SDKs, you can send HTTP or gRPC requests to those endpoints. 
+
+## Next steps
+
+Check the [Hello Kubernetes with Dapr Shared](https://github.com/dapr/dapr-shared/blob/main/docs/tutorial/README.md) tutorial.
```

Comment:
```suggestion
- Try the [Hello Kubernetes tutorial with Dapr Shared](https://github.com/dapr/dapr-shared/blob/main/docs/tutorial/README.md).
- Read more in the [Dapr Shared repo](https://github.com/dapr/dapr-shared/blob/main/README.md)
```

Reply:
I would have thought that either here or better still in the https://github.com/dapr/quickstarts/pull/1030 PR it would have included these instructions https://github.com/dapr/dapr-shared/blob/main/tutorial/README.md#installing-dapr-shared-and-all-applications to install dapr Shared? Can you include these as as heading in the https://github.com/dapr/dapr-shared/blob/main/tutorial/README.md#installing-dapr-shared-and-all-applications PR?

Reply:
I think you are missing this PR: https://github.com/dapr/dapr-shared/pull/52